src/test/java/uk/gov/register/service/ItemValidatorTest.java

```diff
@@ -102,9 +112,42 @@ public void validateItem_throwsValidationException_givenFieldWithCardinalityMany
     }
 
     @Test
-    public void noErrorWhenEntryIsValid() throws IOException, ItemValidationException {
+    public void validateItemIsCanonicalized_throwsValidationException_whenItemIsNotCanonicalized() throws IOException {
+        String jsonString = "{\"text\":\"some text\",\"register\":\"aregister\"}";
+        JsonNode jsonNode = nodeOf(jsonString);
+        Item item = new Item(jsonNode, jsonString);
+
+        try {
+            itemValidator.validateItem("register", item);
+            fail("Entry is not canonicalized but threw no exception");
+        } catch (ItemValidationException e) {
+            assertThat(e.getMessage(), equalTo("Entry is not canonicalized"));
+        }
+    }
+
+    @Test
+    public void validateItemIsCanonicalized_throwsNoValidationException_whenItemIsCanonicalized() throws IOException {
```

Comment:
This test should technically be covered by the below test (`noErrorWhenEntryIsValid`) and can be removed.